docs/extensions/HyperlinkExtensions.md

```diff
@@ -7,36 +7,26 @@ keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp
 
 # HyperlinkExtensions
 
-The [HyperlinkExtension](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.extensions.hyperlink) allows for a Hyperlink element to invoke the execute method on a bound [ICommand](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Input.ICommand) instance when clicked.
-
-## Example
-
-```xaml
-// Use Hyperlink in a wrapped TextBlock with text either side and ensure it executes a command when
-// clicked passing the current data context as the command parameter.
-<TextBlock>
-	<Run>Some leading text with a</Run>
-		<Hyperlink extensions:Hyperlink.Command="{Binding HyperlinkClicked}"
-				extensions:Hyperlink.CommandParameter="{Binding}">hyperlink</Hyperlink>
-	<Run>in the middle.</Run>
-</TextBlock>
-```
-
-## Properties
+The [HyperlinkExtensions](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.hyperlinkextensions) class allows for a [`Hyperlink`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Documents.Hyperlink) element to invoke the execute method on a bound [`ICommand`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Input.ICommand) instance when clicked.
 
-| Property | Description |
-| --| -- |
-| Command | Attached `DependencyProperty` for binding an ICommand instance to a Hyperlink |
-| CommandParameter | Attached `DependencyProperty` for binding a command parameter to a Hyperlink |
+> **Platform APIs:** [HyperlinkExtensions](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.hyperlinkextensions)
 
-## Requirements
+## How it works
 
-| Device family | Universal, 10.0.16299.0 or higher |
-| --- | --- |
-| Namespace | Microsoft.Toolkit.Uwp.Extensions |
-| NuGet package | [Microsoft.Toolkit.Uwp.UI](https://www.nuget.org/packages/Microsoft.Toolkit.Uwp.UI/) |
+Using the `HyperlinkExtensions` attached properties simply requires you to assign them to a `Hyperlink` control in XAML, and bind to an `ICommand` instance in a viewmodel. Both `{Binding}` and `{x:Bind}` modes are supported, and you can freely choose which one to use depending on your scenario. Here is a small example of these attached properties in action:
 
-## API
+```xml
+<!-- Use Hyperlink in a wrapped TextBlock with text either side and ensure it executes a
+     command when clicked, passing the current data context as the command parameter -->
+<TextBlock xmlns:ui="using:Microsoft.Toolkit.Uwp.UI">
+    <Run>Some leading text with a</Run>
+        <Hyperlink
+            ui:Hyperlink.Command="{Binding HyperlinkClicked}"

```

Comment:
Is this `ui:HyperlinkExtensions.Command` now?

Reply:
Right, that was a leftover from the previous docs page, thanks!
Fixed in f86250396205a05fa7da6098196540addda25a13.